src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx

```diff
@@ -21,6 +22,18 @@ export function VideoEmbedInnerNative() {
   const player = useVideoPlayer()
   const ref = useRef<VideoView>(null)
   const isScreenFocused = useIsFocused()
+  const isAppFocused = useAppState()
+  const [prevFocused, setPrevFocused] = useState(isAppFocused)
+
+  // resume video when coming back from background
+  useEffect(() => {
+    if (isAppFocused !== prevFocused) {
+      setPrevFocused(isAppFocused)
+      if (isAppFocused === 'active') {
+        player.play()
+      }
+    }
```

Comment:
Might be wrong, but is there a reason that we need to store the previous state? Seems that we might be able to just do

```tsx
const appState = useAppState()

useEffect(() => {
  if (appState !== 'active') return
  player.play()
}, [appState, player])
```

Seems that `player` should be stable so the only thing that should change here is `appState`.

Reply:
I wanted to constrain when it fires specifically to when it's going from !active -> active. Your version will also fire on initial render. idk, do you think that matters?

Reply:
Ahh, right. Maybe let's just stick the prev value in a ref then?

Reply:
Yeah that'd work